src/Mesh/babylon.abstractMesh.ts

```diff
@@ -637,6 +637,26 @@
             }
             return this;
         }
+        
+        /**
+         * Return the point Y of a coordinate on a mesh 
+         * @param {number} x
+         * @param {number} z
+         * @param {number} maxHeight (default 200)
+         */
+        public getHeightMeshAtCoordinates(x: number, z: number, maxHeight = 200): Vector3 {		
+            var ray = new BABYLON.Ray(new BABYLON.Vector3(x, maxHeight, z), new BABYLON.Vector3(0, -1, 0), 2*maxHeight);
+            var res = this.intersects(ray, true);

```

Comment:
Do you think this must be in the framework? It depends a lot on the mesh. While it makes sense for the GroundMesh, I'm not sure this is useful for the abstractmesh